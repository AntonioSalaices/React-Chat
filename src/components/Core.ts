export * from './basics/Card/Card';
export * from './basics/Input/Input';
export * from './basics/Message/Message';
export * from './basics/Spinner/Spinner';
export * from './composed/Container/Container';
export * from './composed/ErrorBoundary/ErrorBoundary';
export * from './composed/ErrorPage/ErrorPage';
export * from './composed/Footer/Footer';
export * from './composed/FormField/FormField';
export * from './composed/Header/Header';
export * from './composed/Layout/Layout';
export * from './composed/List/List';
export * from './composed/NotFound/NotFound';
export * from './composed/Section/Section';
export * from './composed/Modal/Modal';
export * from './composed/LazyComponent/LazyComponent';
