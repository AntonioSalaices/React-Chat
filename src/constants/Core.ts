export * from './authConstants';
export * from './eventConstants';
export * from './htmlConstants';
export * from './icons';
export * from './languageConstans';
export * from './routesConstants';
export * from './screenConstants';
export * from './securityConstants';
export * from './Theme';
export * from './serviceConstants';
