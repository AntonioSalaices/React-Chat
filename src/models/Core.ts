export * from './Store.ts';
export * from './Pokemon.ts';
export * from './User.ts';
