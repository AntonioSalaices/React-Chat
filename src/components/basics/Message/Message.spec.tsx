import { RenderResult, render } from '@testing-library/react';
import { Message } from '@Components/Core';

describe('<Message />', () => {
  let component: RenderResult;

  beforeEach(() => {
    component = render(<Message text="simple test message" />);
  });
  test('should render correctly', () => {
    const { container } = component;

    expect(container).toMatchSnapshot();
  });
});
