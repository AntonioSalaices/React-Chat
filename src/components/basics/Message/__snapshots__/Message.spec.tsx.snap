// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Message /> should render correctly 1`] = `
<div>
  <div>
    <h1
      class="text-primary font-bold font-size-lg"
    >
      simple test message
    </h1>
  </div>
</div>
`;
